app.css

```diff
@@ -18,6 +18,8 @@ body{
     clear: both;
 }
 .right { float: right; }
+.red{color:#FF5049 !important;}
+.red-focus{border: 1px solid #FF5049 !important;}
```

Comment:
Please be consistent in code style. There is a space between class name and curly braces while the new changes are missing this code style. Either match the above one to the one with new changes or make the change in new changes.